File: back/src/main/java/br/eng/eliseu/presente/model/filter/EventoPessoaFilter.java
package br.eng.eliseu.presente.model.filter;

import br.eng.eliseu.presente.model.core.BaseFilter;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class EventoPessoaFilter extends BaseFilter {

    private Long id;

    private Long eventoId;

    private Long pessoaId;
    private String pessoaNome;
    private String pessoaCpf;
    private String pessoaEmail;
    private String pessoaTelefone;

}

File: front/src/app/shared/model/filter/evento-pessoa-filter.ts
import {BaseFilter} from '@/shared/model/core/base-filter';

export class EventoPessoaFilter extends BaseFilter {

    id?: number;

    eventoId?: number;

    pessoaId?: number;
    pessoaNome?: string;
    pessoaCpf?: string;
    pessoaEmail?: string;
    pessoaTelefone?: string;

    jaEscolheu: boolean | null = null;
    temLinkMagico: boolean | null = null;

    constructor(init?: Partial<EventoPessoaFilter>) {
        super();
        Object.assign(this, init);
    }
}
